<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-1GSEJ22F" failures="0" tests="16" name="CyberSuccess1.LambdaTest_Automation.SignTest" time="-1663601664.000" errors="1" timestamp="2022-09-19T21:02:24 IST" skipped="15">
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="uncheckedTerms" time="0.000">
    <skipped/>
  </testcase> <!-- uncheckedTerms -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="invalidEmailTest" time="0.000">
    <skipped/>
  </testcase> <!-- invalidEmailTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="emptyNameTest" time="0.000">
    <skipped/>
  </testcase> <!-- emptyNameTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="privacyPolicyRedirectionTest" time="0.000">
    <skipped/>
  </testcase> <!-- privacyPolicyRedirectionTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="verifyElemntsOnPageTest" time="0.000">
    <skipped/>
  </testcase> <!-- verifyElemntsOnPageTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="emptyPhoneTest" time="0.000">
    <skipped/>
  </testcase> <!-- emptyPhoneTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="loginRedirectionTest" time="0.000">
    <skipped/>
  </testcase> <!-- loginRedirectionTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="validRegistrationTest" time="0.000">
    <skipped/>
  </testcase> <!-- validRegistrationTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="emptyCompanyNameTest" time="0.000">
    <skipped/>
  </testcase> <!-- emptyCompanyNameTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="emptyPasswordTest" time="0.000">
    <skipped/>
  </testcase> <!-- emptyPasswordTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="termsRedirectionTest" time="0.000">
    <skipped/>
  </testcase> <!-- termsRedirectionTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="landingPageRedirectionTest" time="0.000">
    <skipped/>
  </testcase> <!-- landingPageRedirectionTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="inValidPhoneTest" time="0.000">
    <skipped/>
  </testcase> <!-- inValidPhoneTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="inValidPasswordTest" time="0.000">
    <skipped/>
  </testcase> <!-- inValidPasswordTest -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="setUp" time="-1663601664.000">
    <error message="
Method setUp requires 2 parameters but 1 were supplied in the BeforeClass annotation." type="org.testng.TestNGException">
      <![CDATA[org.testng.TestNGException: 
Method setUp requires 2 parameters but 1 were supplied in the BeforeClass annotation.
at org.testng.internal.Parameters.checkParameterTypes(Parameters.java:447)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:353)
at org.testng.internal.Parameters.createParameters(Parameters.java:700)
at org.testng.internal.Parameters.createConfigurationParameters(Parameters.java:168)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:302)
at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:176)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:122)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- setUp -->
  <system-out/>
  <testcase classname="CyberSuccess1.LambdaTest_Automation.SignTest" name="emptyEmailTest" time="0.000">
    <skipped/>
  </testcase> <!-- emptyEmailTest -->
  <system-out/>
</testsuite> <!-- CyberSuccess1.LambdaTest_Automation.SignTest -->
